actors/evm/src/interpreter/execution.rs

```diff
@@ -154,13 +154,11 @@ impl<'r, 'a, RT: Runtime + 'r> Machine<'r, 'a, RT> {
     }
 
     pub fn execute(&mut self) -> Result<(), StatusCode> {
-        loop {
-            if self.pc >= self.bytecode.len() {
-                break;
+        while self.pc < self.bytecode.len() {
+            unsafe {
+                self.step();
             }
 
-            self.step();
-
             if self.exit.is_some() {
                 break;
             }
```

Comment:
Why not put this at the top?

Reply:
because it is guaranteed to be None at first execution; saves an unnecessary check at first iteration!

Reply:
That... is pointless. I think we need a better test case.

Reply:
I know, i am horrible, but i cant help myself.

It probably makes 0 diff in gas, but it is a reduntant check damn it!!! :)

Yes for better test cases, lrts start collecting these puppies.